src/components/grid/grid.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import { Flex, Box } from 'grid-styled'
+import styled from 'styled-components'
+
+const Container = styled(Box)`
+
+  @media (min-width: 375px) {
+    min-width: 335px;
```

Comment:
if we're going to be using these as our media breakpoints throughout the sites, then should we place them in the theme file?